proto/api/users.proto

```diff
@@ -36,6 +39,19 @@ message GetUsersResponse {
     repeated User users = 1;
 }
 
+message PostUsersRequest {
+    string username = 2;      
+    string first_name = 4;
+    string last_name = 5;
+    string password = 6;
+}
+  
+message PostUsersResponse {
+    string username = 2;      
+    string first_name = 4;
+    string last_name = 5;
+}
```

Comment:
what is missing from this response that is detrimental?

Reply:
Ah definitely the id but I believe the avatar url should be in here as well